diff --git a/node_modules/@axe-core/puppeteer/.DS_Store b/node_modules/@axe-core/puppeteer/.DS_Store
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/@axe-core/puppeteer/dist/index.js b/node_modules/@axe-core/puppeteer/dist/index.js
index 84a4a52..089cb41 100644
--- a/node_modules/@axe-core/puppeteer/dist/index.js
+++ b/node_modules/@axe-core/puppeteer/dist/index.js
@@ -252,6 +252,8 @@ async function getChildFrame(frame, childSelector) {
   return await frameElm.asElement()?.contentFrame() || null;
 }
 async function assertFrameReady(frame) {
+  // If the CPU is overloaded, the frame.evaluate() function will fail to run before the timeout causes the validation to be rejected.
+  return;
   try {
     const timeoutPromise = new Promise((resolve, reject) => {
       setTimeout(() => {
