diff --git a/node_modules/@crawlee/core/session_pool/session.d.ts b/node_modules/@crawlee/core/session_pool/session.d.ts
index da49c3c8..5343906c 100644
--- a/node_modules/@crawlee/core/session_pool/session.d.ts
+++ b/node_modules/@crawlee/core/session_pool/session.d.ts
@@ -1,5 +1,5 @@
 import type { Cookie as CookieObject, Dictionary } from '@crawlee/types';
-import type { Cookie, SerializedCookieJar } from 'tough-cookie';
+import type { Cookie } from 'tough-cookie';
 import { CookieJar } from 'tough-cookie';
 import type { Log } from '@apify/log';
 import type { ResponseLike } from '../cookie_utils';
@@ -8,7 +8,7 @@ import type { ResponseLike } from '../cookie_utils';
  */
 export interface SessionState {
     id: string;
-    cookieJar: SerializedCookieJar;
+    cookieJar: any;
     userData: object;
     errorScore: number;
     maxErrorScore: number;
