diff --git a/node_modules/got-scraping/node_modules/header-generator/.DS_Store b/node_modules/got-scraping/node_modules/header-generator/.DS_Store
new file mode 100644
index 0000000..6406205
Binary files /dev/null and b/node_modules/got-scraping/node_modules/header-generator/.DS_Store differ
diff --git a/node_modules/got-scraping/node_modules/header-generator/header-generator.js b/node_modules/got-scraping/node_modules/header-generator/header-generator.js
index 0c7779f..012cdfc 100644
--- a/node_modules/got-scraping/node_modules/header-generator/header-generator.js
+++ b/node_modules/got-scraping/node_modules/header-generator/header-generator.js
@@ -91,16 +91,22 @@ class HeaderGenerator {
             strict,
         };
         this.uniqueBrowsers = [];
-        this.headersOrder = JSON.parse((0, fs_1.readFileSync)(`${__dirname}/data_files/headers-order.json`).toString());
-        const uniqueBrowserStrings = JSON.parse((0, fs_1.readFileSync)(`${__dirname}/data_files/browser-helper-file.json`, 'utf8').toString());
+        // Remove unnecessary dependencies to package modules using webpack.
+        // this.headersOrder = JSON.parse((0, fs_1.readFileSync)(`${__dirname}/data_files/headers-order.json`).toString());
+        // const uniqueBrowserStrings = JSON.parse((0, fs_1.readFileSync)(`${__dirname}/data_files/browser-helper-file.json`, 'utf8').toString());
+        this.headersOrder = {};
+        const uniqueBrowserStrings = [];
         for (const browserString of uniqueBrowserStrings) {
             // There are headers without user agents in the datasets we used to configure the generator. They should be disregarded.
             if (browserString !== constants_1.MISSING_VALUE_DATASET_TOKEN) {
                 this.uniqueBrowsers.push(this.prepareHttpBrowserObject(browserString));
             }
         }
-        this.inputGeneratorNetwork = new generative_bayesian_network_1.BayesianNetwork({ path: `${__dirname}/data_files/input-network-definition.zip` });
-        this.headerGeneratorNetwork = new generative_bayesian_network_1.BayesianNetwork({ path: `${__dirname}/data_files/header-network-definition.zip` });
+        // Remove unnecessary dependencies to package modules using webpack.
+        // this.inputGeneratorNetwork = new generative_bayesian_network_1.BayesianNetwork({ path: `${__dirname}/data_files/input-network-definition.zip` });
+        // this.headerGeneratorNetwork = new generative_bayesian_network_1.BayesianNetwork({ path: `${__dirname}/data_files/header-network-definition.zip` });
+        this.inputGeneratorNetwork = {};
+        this.headerGeneratorNetwork = {};
     }
     /**
     * Generates a single set of ordered headers using a combination of the default options specified in the constructor
