diff --git a/node_modules/puppeteer/node_modules/puppeteer-core/lib/cjs/puppeteer/node/ChromeLauncher.js b/node_modules/puppeteer/node_modules/puppeteer-core/lib/cjs/puppeteer/node/ChromeLauncher.js
index a66623a..19d93f8 100644
--- a/node_modules/puppeteer/node_modules/puppeteer-core/lib/cjs/puppeteer/node/ChromeLauncher.js
+++ b/node_modules/puppeteer/node_modules/puppeteer-core/lib/cjs/puppeteer/node/ChromeLauncher.js
@@ -126,7 +126,7 @@ class ChromeLauncher extends BrowserLauncher_js_1.BrowserLauncher {
             // AcceptCHFrame disabled because of crbug.com/1348106.
             'AcceptCHFrame',
             'MediaRouter',
-            'OptimizationHints',
+            // 'OptimizationHints', // Disabled due to browser instability
             ...(turnOnExperimentalFeaturesForTesting
                 ? []
                 : [
